### PR TITLE
Paywalls: Improve Chinese localization

### PR DESCRIPTION
**Pull request by aboedo**
Improvements to Chinese Traditional and Chinese Simplified localizations based on user feedback.

### CODE REVIEW

**Comment by francisfeng** [comment]
Hi Andy. This is Francis Feng.

> In the localization strings it looked like "个" would only show up for subscriptions that were 2, 3 or 6 months long. Is that not the case for you? 

No, my app only has a monthly plan (no plans for 2, 3, 6 months). But the paywall shows up as “¥15/个月”.

**Review by tonidero — APPROVED** [review]
I don't know chinese 😅 but once we've confirmed the changes :shipit:

**Comment by aboedo** [comment]
@francisfeng thanks so much again for opening up the PR and bringing this up in the first place!

**Comment by aboedo** [comment]
I'm not sure how we're getting the `个` in monthly, but I'll try to dig into it so that we can avoid showing it

**Comment by francisfeng** [comment]
> I'm not sure how we're getting the `个` in monthly, but I'll try to dig into it so that we can avoid showing it

It comes from the variable `{{ price_per_period }}`, which is not localized in the package, as I didn’t find the localized string.

{{ price_per_period }} => ¥15.00/个月

File: RevenueCatUI/Resources/zh_TW.lproj/Localizable.strings [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 "3 Month" = "3個月";
 "2 Month" = "2個月";
 "Monthly" = "每月";
 "Weekly" = "每週";
-"Lifetime" = "壽命";
-"%d%% off" = "%d%%折";
+"Lifetime" = "永久";
+"%d%% off" = "優惠%d%%";
 "Continue" = "繼續";
-"Default_offer_details_with_intro_offer" = "開始 {{ sub_offer_duration }} 試用，然後 {{total_price_and_per_month }}。";
```
Comment (NachoSoto) [inline RevenueCatUI/Resources/zh_TW.lproj/Localizable.strings:19]: Ouch

**Review by NachoSoto — APPROVED** [review]
🙌🏻

File: RevenueCatUI/Resources/zh_HK.lproj/Localizable.strings [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 "2 Month" = "2個月";
 "Monthly" = "每月";
 "Weekly" = "每週";
-"Lifetime" = "壽命";
-"%d%% off" = "%d%%折";
+"Lifetime" = "永久";
+"%d%% off" = "優惠%d%%";
 "Continue" = "繼續";
-"Default_offer_details_with_intro_offer" = "開始 {{ sub_offer_duration }} 試用，然後 {{total_price_and_per_month }}。";
+"Default_offer_details_with_intro_offer" = "開始{{ sub_offer_duration }}試用，然後{{ total_price_and_per_month }}。";
```
Comment (greenietea) [inline RevenueCatUI/Resources/zh_HK.lproj/Localizable.strings:19]: This reads weird in Chinese. I would go with

```suggestion
"Default_offer_details_with_intro_offer" = "試用期{{ sub_offer_duration }}，屆滿後{{ total_price_and_per_month }}。";
```

File: RevenueCatUI/Resources/zh-Hans.lproj/Localizable.strings [hunk 3]
```diff
@@ -5,15 +5,15 @@
...
 "2 Month" = "2个月";
 "Monthly" = "每月";
 "Weekly" = "每周";
-"Lifetime" = "寿命";
-"%d%% off" = "%d%%折";
+"Lifetime" = "永久";
+"%d%% off" = "优惠%d%%";
 "Continue" = "继续";
-"Default_offer_details_with_intro_offer" = "开始 {{ sub_offer_duration }} 试用，然后 {{total_price_and_per_month }}。";
+"Default_offer_details_with_intro_offer" = "开始{{ sub_offer_duration }}试用，然后{{ total_price_and_per_month }}。";
```
Comment (greenietea) [inline RevenueCatUI/Resources/zh-Hans.lproj/Localizable.strings:19]:
```suggestion
"Default_offer_details_with_intro_offer" = "试用期{{ sub_offer_duration }}，届满后{{ total_price_and_per_month }}。";
```

File: RevenueCatUI/Resources/zh-Hans.lproj/Localizable.strings [hunk 4]
```diff
@@ -5,15 +5,15 @@
...
-"Annual" = "年度的";
+"Annual" = "每年";
 "6 Month" = "6个月";
 "3 Month" = "3个月";
 "2 Month" = "2个月";
 "Monthly" = "每月";
 "Weekly" = "每周";
-"Lifetime" = "寿命";
-"%d%% off" = "%d%%折";
+"Lifetime" = "永久";
```
Comment (greenietea) [inline RevenueCatUI/Resources/zh-Hans.lproj/Localizable.strings:16]:
```suggestion
"Lifetime" = "終身";
```

File: RevenueCatUI/Resources/zh_HK.lproj/Localizable.strings [hunk 5]
```diff
@@ -1,19 +1,19 @@
...
-"Annual" = "年度的";
+"Annual" = "每年";
 "6 Month" = "6個月";
 "3 Month" = "3個月";
 "2 Month" = "2個月";
 "Monthly" = "每月";
 "Weekly" = "每週";
-"Lifetime" = "壽命";
-"%d%% off" = "%d%%折";
+"Lifetime" = "永久";
```
Comment (greenietea) [inline RevenueCatUI/Resources/zh_HK.lproj/Localizable.strings:16]:
```suggestion
"Lifetime" = "終身";
```

File: RevenueCatUI/Resources/zh_TW.lproj/Localizable.strings [hunk 6]
```diff
@@ -1,19 +1,19 @@
...
-"Annual" = "年度的";
+"Annual" = "每年";
 "6 Month" = "6個月";
 "3 Month" = "3個月";
 "2 Month" = "2個月";
 "Monthly" = "每月";
 "Weekly" = "每週";
-"Lifetime" = "壽命";
-"%d%% off" = "%d%%折";
+"Lifetime" = "永久";
```
Comment (greenietea) [inline RevenueCatUI/Resources/zh_TW.lproj/Localizable.strings:16]:
```suggestion
"Lifetime" = "終身";
```

File: RevenueCatUI/Resources/zh_TW.lproj/Localizable.strings [hunk 7]
```diff
@@ -1,19 +1,19 @@
...
+"%d%% off" = "優惠%d%%";
 "Continue" = "繼續";
-"Default_offer_details_with_intro_offer" = "開始 {{ sub_offer_duration }} 試用，然後 {{total_price_and_per_month }}。";
+"Default_offer_details_with_intro_offer" = "開始{{ sub_offer_duration }}試用，然後{{ total_price_and_per_month }}。";
```
Comment (greenietea) [inline RevenueCatUI/Resources/zh_TW.lproj/Localizable.strings:19]:
```suggestion
"Default_offer_details_with_intro_offer" = "試用期{{ sub_offer_duration }}，屆滿後{{ total_price_and_per_month }}。";
```

**Review by greenietea — APPROVED** [review]

**Comment by aboedo** [comment]
@greenietea thanks for the suggestions!! 🙌

**Pull request merged**